mlflow/utils/databricks_utils.py

```diff
@@ -151,6 +151,16 @@ def get_cluster_id():
     return spark_session.conf.get("spark.databricks.clusterUsageTags.clusterId")
 
 
+def get_job_group_id():
+    try:
+        dbutils = _get_dbutils()
+        job_group_id = dbutils.entry_point.getJobGroupId()
```

Comment:
I found this method in the dbutils.py file. Let me know if this is sufficient to cover all environments.

Reply:
From offline discussion, this seems like the right method to use / should be allowlisted in all cluster environments! Ideally we can cover this our tests too